Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -3339,6 +3360,26 @@ ErrorOr<RefPtr<StyleValue>> Parser::parse_dynamic_value(ComponentValue const& co
             // Declarations using `var()` should already be parsed as an UnresolvedStyleValue before this point.
             VERIFY_NOT_REACHED();
         }
+
+        if (function.name().equals_ignoring_ascii_case("min"sv)) {
+            auto parameters = TRY(parse_comma_separated_function_parameters(function.values()));
```

Comment:
We already have `parse_a_comma_separated_list_of_component_values()` which I think would be a better fit here. It'll give you a Vector of ComponentValue Vectors, one inner Vector per parameter.